compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/Types.java

```diff
@@ -6067,6 +6067,10 @@ BType getTypeWithoutNil(BType type) {
         return BUnionType.create(null, new LinkedHashSet<>(nonNilTypes));
     }
 
+    public boolean isFixedLengthTuple(BTupleType bTupleType) {
+        return bTupleType.restType == null || isNeverTypeOrStructureTypeWithARequiredNeverMember(bTupleType.restType);
```

Comment:
Isn't the null check already being handled within `isNeverTypeOrStructureTypeWithARequiredNeverMember ()`?

Reply:
https://github.com/ballerina-platform/ballerina-lang/blob/856f3176aa9898febb2de7505d240f735571edbe/compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/Types.java#L6067-L6069

It returns `false` for `null`. But we need `true` for `null` here right?